src/main/java/com/google/sps/data/OrganizationUpdater.java

```diff
@@ -59,6 +59,7 @@ public void updateOrganization(HttpServletRequest request, GivrUser user, boolea
     properties.put("org-description","orgDescription");
     properties.put("approval", "isApproved");
     properties.put("moderator-list", "moderatorList");
+    properties.put("org-resource-category", "resourceCategories");
```

Comment:
you only made sure that adding an organization works but you also must modify the edit page too i.e now only have that area but also prefill the values in the form. Can you also fix this for zip code because the edit page is also failing because of that

Reply:
please test it when you're done too haha

Reply:
I left the updating of the edit page as a TODO, because on master/my branches it still looks like the edit page is inaccessible because of those bugs from before where it just shows the help near me and register orgs page. How can I access the page that shows the organizations so I can edit them?

Reply:
resolved and tested